hack/verify-crd-schema-checker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+# Build crd-schema-checker when it's not present and not overriden for a specific file.
+if [ -z "${CRD_SCHEMA_CHECKER:-}" ];then
+  ${TOOLS_MAKE} crd-schema-checker
+  CRD_SCHEMA_CHECKER="${TOOLS_OUTPUT}/crd-schema-checker"
+fi
+
+BASE_SHA = ${PULL_BASE_SHA:-master}
+for f in **/*.crd.yaml; do
+  git show $BASE_SHA:$f > $f.base
+  crd-schema-checker check-manifests --existing-crd-filename $f.base --new-crd-filename $f
```

Comment:
if this exits non-zero, what cleans up the file?  perhaps write a temp file instead of $f.base?